src/Pact/Gas/Table.hs

```diff
@@ -0,0 +1,173 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE LambdaCase #-}
+module Pact.Gas.Table where
+
+import qualified Data.Map as Map
+
+import Pact.Types.Gas
```

Comment:
import formatting

Reply:
Oh, do you guys do the spacing for qualified? (That style is banned over here.) I can switch it...

Reply:
`Pact.Types.Gas` should be grouped with the rest of the `Pact` imports. We don't usually have spaced qualified's, just like you guys.

Reply:
ahhhh